src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -113,36 +123,33 @@ const GuideDrawer = createReactClass({
         <StyledContent>
           <div
             dangerouslySetInnerHTML={{
-              __html: this.interpolate(
-                currentGuide.steps[currentStep - 1].message,
-                messageVariables
-              ),
+              __html: this.interpolate(guide.steps[step - 1].message, messageVariables),
             }}
           />
           <div style={{marginTop: '1em'}}>
-            {currentStep < currentGuide.steps.length ? (
+            {step < guide.steps.length ? (
               <div>
                 <Button priority="success" size="small" onClick={nextStep}>
                   {t('Next')} &rarr;
                 </Button>
               </div>
             ) : (
               <div style={{textAlign: 'center'}}>
-                <p>{t('Did you find this guide useful?')}</p>
+                {!isTip && <p>{t('Did you find this guide useful?')}</p>}
                 <Button
                   priority="success"
                   size="small"
                   onClick={() => this.handleFinish(true)}
                 >
-                  {t('Yes')} &nbsp; &#x2714;
+                  {isTip ? guide.cta_text : <span>{t('Yes')} &nbsp; &#x2714;</span>}
                 </Button>
                 <Button
                   priority="success"
                   size="small"
                   style={{marginLeft: '0.25em'}}
                   onClick={() => this.handleFinish(false)}
                 >
-                  {t('No')} &nbsp; &#x2716;
+                  {isTip ? t('Dismiss') : <span>{t('No')} &nbsp; &#x2716;</span>}
```

Comment:
This `dismiss` cta is a bit confusing. I thought it was analogous to `dismiss` in the general context of the drawer so I was expecting to see it show up as dismissed in the backend not viewed. I think maybe because it's not really dismissable in the former sense and also the tips target is present on both pages, when you navigate to the issue tour and add '#assistant' to the url, the tip wins out when the intended behavior would be to trigger the 'issue tour' (this happens if you haven't dismissed the issue tour).

Reply:
Gotcha - that makes sense. They were overlapping because I fiddled with the threshold so it should be fine for a bit.